src/ast/ast_build_ar_exp.c

```diff
@@ -0,0 +1,328 @@
+/*
+* Copyright 2018-2019 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#include "ast_build_ar_exp.h"
+#include "../util/rmalloc.h"
+#include "../arithmetic/funcs.h"
+#include <assert.h>
+
+static const char *_ASTOpToString(AST_Operator op) {
+	switch(op) {
+	case OP_PLUS:
+		return "ADD";
```

Comment:
All of the operators have strings in the parser, like https://github.com/RedisGraph/RedisGraph/blob/master/deps/libcypher-parser/lib/src/operators.c#L139, but there's not an interface to access them, which is rather frustrating.

Reply:
Take a note, maybe we should extend the parser interface.